torch/_dynamo/variables/torch.py

```diff
@@ -855,12 +855,20 @@ def speculate_subgraph(f, sub_args, graph_checkpoint, checkpoint):
                 # NB: we don't bother populating graphargs, as
                 # they won't actually get used by anything
 
-            output = f.call_function(tx, args, {})
+            try:
+                output = f.call_function(tx, args, {})
+            except Exception:
+                raise
```

Comment:
This try-except seems pretty pointless?

Reply:
I understand that python can implicitly propagate exceptions up to the higher layer, but I do also see dynamo code where we make the propagation explicitly like in c++ for readability I guess. Will revert if not preferred.